app/styles/_notifications.less

```diff
@@ -0,0 +1,44 @@
+.drawer-pf-notification:hover {
+  cursor: pointer;
```

Comment:
This seems like a sign of an accessibility problem with the notification drawer itself.